x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityCrossClusterApiKeySigningIT.java

```diff
@@ -318,6 +336,62 @@ private Response performRequestWithRemoteAccessUser(final Request request) throw
         return client().performRequest(request);
     }
 
+    private String extractJsonValue(String jsonLine, String fieldName) {
```

Comment:
Instead of this custom json logic I think it would be cleaner to use [XContentHelper.convertToMap](https://github.com/elastic/elasticsearch/blob/main/server/src/main/java/org/elasticsearch/common/xcontent/XContentHelper.java#L239) and also the constants from [here](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/AuditLevel.java#L14-L29). It will make the test easier to maintain.